ci-operator/jobs/operator-framework/operator-sdk/operator-framework-operator-sdk-master-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
+  operator-framework/operator-sdk:
+  - agent: kubernetes
+    branches:
+    - ^master$
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: branch-ci-operator-framework-operator-sdk-master-images
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
```

Comment:
Curious is there a way to give everyone in the [OWNERS](https://github.com/operator-framework/operator-sdk/blob/master/OWNERS) file access to the namespace as well? This would help debug failing tests, we ran into this problem last week when a user's test was failing and it would have been useful then. cc @shawn-hurley

Reply:
I don't think this is possible currently from the test platform, the user is an admin though so the author should be able to grant access.